ubenchmark/src/main/java/org/postgresql/benchmark/escaping/EscapeProcessing.java

```diff
@@ -0,0 +1,20 @@
+package org.postgresql.benchmark.escaping;
+
+import org.openjdk.jmh.annotations.*;
+import org.postgresql.core.Parser;
+
+import java.util.concurrent.TimeUnit;
+
+@Measurement(iterations = 10, time = 1, timeUnit = TimeUnit.SECONDS)
+@Warmup(iterations = 10, time = 1, timeUnit = TimeUnit.SECONDS)
+@State(Scope.Thread)
+@Threads(1)
+@BenchmarkMode(Mode.AverageTime)
+@OutputTimeUnit(TimeUnit.NANOSECONDS)
+public class EscapeProcessing {
+
+    @Benchmark
+    public void escapeFunctionWithDate() throws Exception {
+        Parser.replaceProcessing("{fn week({d '2005-01-24'})}", true, false);
```

Comment:
1) Use fields for arguments: http://hg.openjdk.java.net/code-tools/jmh/file/7a25c71b43bf/jmh-samples/src/main/java/org/openjdk/jmh/samples/JMHSample_10_ConstantFold.java#l74
2) return the value of `Parser.replaceProcessing`: http://hg.openjdk.java.net/code-tools/jmh/file/7a25c71b43bf/jmh-samples/src/main/java/org/openjdk/jmh/samples/JMHSample_08_DeadCode.java#l67